kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/CronJobTest.java

```diff
@@ -56,15 +56,15 @@ void testList() {
       .addNewItem()
       .and().build()).once();
 
-    CronJobList cronJobList = client.batch().cronjobs().list();
+    CronJobList cronJobList = client.batch().v1().cronjobs().list();
```

Comment:
You might need to change import for CronJobList here, or maybe add a separate test(V1CronJobTest). You would need to modify mockServer expectations `server.expect().withPath(..)` to use `v1` versions in URLs